src/hooks/useHandRaising.js

```diff
@@ -0,0 +1,50 @@
+import { useCallback } from 'react';
+import { useDaily, useParticipantIds } from '@daily-co/daily-react-hooks';
+export const HAND_RAISED_USER_DATA_KEY = 'hr'; // keep this as small as possible: the max payload size is 4K characters
```

Comment:
```suggestion
export const HAND_RAISED_USER_DATA_KEY = 'hr'; // Keep this as small as possible: the max payload size is 4K characters
```

(ok I realize this is super nitpick, but we use capitalized comments in the rest of the file and it looks a bit neater :))